AirCasting/SessionViews/StandaloneSessionCardView.swift

```diff
@@ -42,10 +42,11 @@ struct StandaloneSessionCardView: View {
     var content: some View {
         VStack(spacing: 15) {
             Text(Strings.StandaloneSessionCardView.heading)
-                .font(Fonts.boldHeading3)
+                .font(Fonts.moderateBoldHeading1)
                 .foregroundColor(.darkBlue)
                 .multilineTextAlignment(.center)
             Text(Strings.StandaloneSessionCardView.description)
+                .font(Fonts.moderateRegularHeading3)
                 .multilineTextAlignment(.center)
             finishAndSyncButton
             finishAndDontSyncButton
```

Comment:
What about the font for the second button? It looks very small and doesn't look like in the designs

Reply:
I made some changes to this screen. Could you please check it because I am unable to test it 😞.